src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/LicenceConditionService.kt

```diff
@@ -208,6 +349,70 @@ class LicenceConditionService(
         BespokeCondition(licence = licenceEntity, conditionSequence = index, conditionText = condition),
       )
     }
+
+    if (newConditions.isNotEmpty()) {
+      val changes = mapOf(
+        "typeOfChange" to "add",
+        "condition" to "bespoke",
+        "changes" to
+          newConditions.map {
+            mapOf(
+              "conditionCode" to "",
+              "conditionType" to "",
+              "conditionText" to it,
+            )
+          },
+      )
+      auditEventRepository.saveAndFlush(
+        AuditEvent(
+          licenceId = licenceId,
+          username = username,
+          fullName = "${createdBy.firstName} ${createdBy.lastName}",
+          summary = "Add bespoke conditions for ${licenceEntity.forename} ${licenceEntity.surname}",
+          detail = "ID ${licenceEntity.id} type ${licenceEntity.typeCode} status ${licenceEntity.statusCode.name} version ${licenceEntity.version}",
+          changes = changes,
+        ),
+      )
+    }
+
+    if (removedConditions.isNotEmpty()) {
+      val changes = mapOf(
+        "typeOfChange" to "delete",
+        "condition" to "bespoke",
+        "changes" to
+          removedConditions.map {
+            mapOf(
+              "conditionCode" to "",
+              "conditionType" to "",
+              "conditionText" to it,
+            )
+          },
+      )
+      auditEventRepository.saveAndFlush(
```

Comment:
So for one call to update bespoke events you could end up with something like:

```yaml
  Event 1: 
       type: Removed bespoke conditions
       changes:
           typeOfChange/condition: delete bespoke conditions
           changes:
                - condition1 code, type text
                - condition2 code, type text
              
  Event 2: 
       type: Added bespoke conditions
       changes:
           typeOfChange/condition: added bespoke conditions
           changes:
                - condition3 code, type text
```

Feels like we could possible compress this structure structure a bit to make it more concise but contain the same info:
```yaml
  Event 1: 
       type: Updated bespoke conditions
       changes:
           changes:
                - condition1, ADDED, code, type, text
                - condition2, ADDED, code, type, text
                - condition3, REMOVED, code, type, text 
```